src/main/resources/static/js/subscription.js

```diff
@@ -459,11 +460,13 @@ jQuery(document).ready(function () {
                     "title": "Action",
                     "data": null,
                     "render": function (data, type, row, meta) {
-
-                        if (isSecured == false || (row.ldapUserName == currentUser && row.ldapUserName != null)) {
+                    	var unsecureEditAll = isSecured == false;
+                    	var securedEditOwn = (row.ldapUserName == currentUser && row.ldapUserName != null);
```

Comment:
Please check the indention here, some lines are indented 2 spaces, some line indented while they should not be.

Reply:
ok